ci-operator/step-registry/baremetal/lab/ipi/conf/load-balancer/baremetal-lab-ipi-conf-load-balancer-commands.sh

```diff
@@ -47,21 +49,24 @@ listen api-server-6443
     bind :::6443
     mode tcp
     server api_vip $API_VIP:6443 check inter 1s
+    server api_vip_v6 [$API_VIP_V6]:6443 check inter 1s
 listen machine-config-server-22623
     bind :::22623
     mode tcp
     server api_vip $API_VIP:22623 check inter 1s
-
+    server api_vip_v6 [$API_VIP_V6]:22623 check inter 1s
 listen ingress-router-80
     bind :::80
     mode tcp
     balance source
     server api_vip $INGRESS_VIP:80 check inter 1s
+    server api_vip_v6 [$INGRESS_VIP_V6]:80 check inter 1s
 listen ingress-router-443
     bind :::443
     mode tcp
     balance source
     server api_vip $INGRESS_VIP:443 check inter 1s
+    server api_vip_v6 [$INGRESS_VIP_V6]:443 check inter 1s
```

Comment:
We should check if ipv4 or v6 is enabled to add all these entries for the load balancer